frontend/src/scenes/billing/Billing.stories.tsx

```diff
@@ -79,7 +79,7 @@ export const BillingUnsubscribeModal = (): JSX.Element => {
         },
     })
 
-    return <UnsubscribeSurveyModal product={billingJson.products[0] as BillingProductV2Type} />
+    return <UnsubscribeSurveyModal product={billingJson.products[0] as unknown as BillingProductV2Type} />
```

Comment:
Might be better to change the file we're importing this fixture from so it exports the JSON as a typed object. 

e.g. 

`
export const billingJSON: BillingProjectV2Type[] = { ... jsonData ... }
`

Reply:
Hmm I can try that

Reply:
ok doing that opens up a can of worms. But when I don't have `as unknown` I get an error with the `headline`, which is really odd because it _does_ exist in the JSON object for this product. And if I remove the `features` array from the object, then it only complains that the `features` array is missing, not that the headline is the wrong type.

```
Conversion of type '{ name: string; description: string; price_description: null; features: ({ key: string; name: string; description: string; images: null; icon_key: null; type: null; } | { key: string; name: string; description: string; images: { ...; }; icon_key: null; type: string; } | { ...; })[]; ... 24 more ...; inclusion_only: ...' to type 'BillingProductV2Type' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type '{ name: string; description: string; price_description: null; usage_key: null; image_url: string; docs_url: string; subscribed: boolean; plans: never[]; type: string; free_allocation: number; tiers: never[]; ... 17 more ...; headline?: undefined; }' is not comparable to type 'BillingProductV2Type'.
    Types of property 'headline' are incompatible.
      Type 'undefined' is not comparable to type 'string'
```

So idk how to fix this if you want to take a look?

Reply:
Ugh this is kind of nasty.

Reply:
It's possible to do but requires some finagling of the object itself because we do dayjs parsing of the response client side. 

The other problem is that you can't strings to enums (e.g. "dashboards" >> AvailableFeature.dashboard). The easiest way around this is to make AvailableFeature a string union. I also had to change LicensePlan to line up better with the types we use in billing.

I've pretty much made most of the changes. I just need convert AvailableFeature over.

Reply:
I also entirely overwrote the json object with one I copied from a local billing-v2 response in the network tab.

Reply:
ok, feel free to push up a fix 😄 Alternately, since all we need is the product here, we can have a new fixture for just the one product. That would require less type conversion.

Reply:
Ok, added a fix